2006Redone Server/src/redone/net/packets/impl/Bank10.java

```diff
@@ -28,14 +28,15 @@ public void processPacket(Client player, int packetType, int packetSize) {
 			player.getPlayerAssistant().useOperate(removeId);
 			break;
 		case 3900:
-			player.getShopAssistant().buyItem(removeId, removeSlot, 5);
+			player.getShopAssistant().buyItem(removeId, removeSlot, 10);
 			break;
 
 		case 3823:
 			if(!player.getItemAssistant().playerHasItem(removeId)) {
 				return;
 			}
-			player.getShopAssistant().sellItem(removeId, removeSlot, 5);
+			player.getShopAssistant().sellItem(removeId, removeSlot, 10);

```

Comment:
Is there any reason to change the 5 option to 10?
I feel like this is just confusing the menu even more.

Reply:
no reason besides the sell 10 option being replaced by sell all

Reply:
Probably better to leave it as `5`?
```suggestion
			player.getShopAssistant().sellItem(removeId, removeSlot, 5);
```

Reply:
yeah maybe it is better